_posts/2018-04-09-Evolving-Embed.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Evolving Embed
+excerpt: What's new with our Embed feature
+layout: post
+date: 2018-02-12 08:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+During last couple months we invested considerable amount of resources to fix most of the pending 
+bugs on Apiary Embed feature. We've also introduced a couple of new features like **multiple Embeds support**,
+**event hooks** and ability to **programmatically destroy Embeds**.
+
+## What is Embed ?
+
+Embed is a simple JavaScript that you place in ordinary HTML page. It shows your API Project
+hosted on [apiary.io](https://apiary.io/) in *IFrame* and allows you to customize it with your brand colors.
+You can then host this HTML page on your own domain. The end result looks like you are hosting 
+your own API Documentation. If you edit your API Project on [Apiary](https://apiary.io) the regenerated
+documentation is instantly displayed on your own domain.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script src="https://api.apiary.io/seeds/embed.js"></script>
+    <script>
+      const embed = new Apiary.Embed({
+        subdomain: 'your-api-project-subdomain',
+        element: '.container',
+        preferences: {
+          permalinks: true,
+        },
+      });
+    </script>
+  </head>
+  <body>
+    <div class="container" style="width:100%; height: 900px"></div>
+  </body>
+</html>
+```
+
+For more information about Embed, please visit our [help page](https://help.apiary.io/tools/embed/).
+
+## Bug fixes
```

Comment:
I think it would be a better idea to show the new features before the bug fixes.

Reply:
Since you've used title case for other headings like "New Features", I'd capitalise "Fixes" here too for consistency.